dc-model/src/main/java/de/digitalcollections/model/MainSubType.java

```diff
@@ -14,7 +14,7 @@ public MainSubType() {}
   public MainSubType(String mainType, String subType) {
```

Comment:
```suggestion
  /**
   * @param mainType main type is mandatory
   * @param subType optional sub type, can be {@code null}
   */
  public MainSubType(String mainType, String subType) {
```
...ums vllt. noch ganz deutlich zu machen, dass es so gewünscht ist.